app/models/collection.rb

```diff
@@ -334,15 +334,16 @@ def prompt_meme?
     return self.challenge_type == "PromptMeme"
   end
 
-  def get_maintainers_email
-    return self.email if !self.email.blank?
-    return parent.email if parent && !parent.email.blank?
-    "#{self.maintainers.collect(&:user).flatten.uniq.collect(&:email).join(',')}"
+  def maintainers_list
+    @maintainers_list = self.maintainers.collect(&:user).flatten.uniq
```

Comment:
This both sets it as an instance variable and returns it. Rubocop complains about an unnecessary return when I return without setting the instance variable.

Reply:
I think rubocop might complain about an extra `return` keyword, since Ruby returns the last statement by default. Hopefully it should be happy with this though:
```suggestion
    self.maintainers.collect(&:user).flatten.uniq
```